### PR TITLE
go: update Go to 1.16.5

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Comment by benma** [comment]
I tested Linux and Android manually, both work.

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The below instructions assume a unix environment.
 
 The following dependencies need to be installed:
 
-- [Go](https://golang.org/doc/install) version 1.14
+- [Go](https://golang.org/doc/install) version 1.16
```
Comment (x1ddos) [inline README.md:50]: docs/BUILD.md also references go1.14.

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module github.com/digitalbitbox/bitbox-wallet-app
 
-go 1.14
+go 1.16
```
Comment (x1ddos) [inline go.mod:3]: Also FYI here: makes sense to bump only if the plan is to use newer features of go introduced in 1.16. https://golang.org/doc/modules/gomod-ref

**Review by x1ddos — APPROVED** [review]
LGTM

File: go.mod (hunk 2 above)
Comment (benma) [reply go.mod:3]: What about potential security updates?

Comment (x1ddos) [reply go.mod:3]: those are covered by patch releases: https://golang.org/doc/devel/release#go1.14.minor

**Pull request merged**